packages/Datadog.Unity/Runtime/WebGL/DatadogWebGLPlatform.cs

```diff
@@ -50,11 +50,12 @@ public DatadogWorker CreateWorker()
 
         public void SetVerbosity(CoreLoggerLevel logLevel)
         {
-
+            // Not implemented on Web
         }
 
         public void SetTrackingConsent(TrackingConsent trackingConsent)
         {
+            DDCore_SetTrackingConsent(trackingConsent.ToWebValue());
```

Comment:
and here it is a right spelling, but different from `DDCore_SetTrackingConent` used above. There is no test checking this call?

Reply:
No unit test no :( I'll go back and add an integration test for tracking consent on all platforms. RUM-9607